sdk/servicebus/azure-servicebus/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,429 @@
+# Troubleshoot Azure Service Bus client library issues
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Service Bus client library for Python.
+
+## Table of contents
+
+* [General troubleshooting](#general-troubleshooting)
+  * [Enable client logging](#enable-client-logging)
+  * [Common exceptions](#common-exceptions)
+  * [Timeouts](#timeouts)
+* [Troubleshooting authentication issues](#troubleshooting-authentication-issues)
```

Comment:
Add a separate section for Threading/Concurrency. This section should explicitly note that the Python ServiceBus SDK is not thread and coroutine safe.

Reply:
This section should also include sync and async samples with locks to show how to call send_messages concurrently, to avoid errors like:
`AttributeError: 'NoneType' object has no attribute 'client_ready_async'`
`azure.servicebus.exceptions.ServiceBusError: Handler failed: 'NoneType' object has no attribute 'create_sender_link'`

Reply:
See the Event Hubs thread safety section in the README: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/eventhub/azure-eventhub#thread-safety